lte/gateway/c/session_manager/SessionStore.h

```diff
@@ -36,9 +36,10 @@ typedef std::unordered_map<
     SessionUpdate;
 
 enum SessionSearchCriteriaType {
-  IMSI_AND_APN        = 0,
-  IMSI_AND_SESSION_ID = 1,
-  IMSI_AND_UE_IPV4    = 2,
+  IMSI_AND_APN              = 0,
+  IMSI_AND_SESSION_ID       = 1,
+  IMSI_AND_UE_IPV4          = 2,
+  IMSI_AND_UE_IPV4_OR_IPV6  = 3,
```

Comment:
this will be use when we receive a rule report from `pipelined`

Either we find the session by its v4 or v6. But the usage will be forwarded to the same session, no matter which ip version we use